### ISSUE TITLE
QA_EXPT-15 & EXPT-16_Introduction and Theory_content-is-same

### ISSUE THREAD

**Issue by BSravanthi** [Virtual-Labs/analog-cmos-vlsi-circuit-design-iiith#45]
Defect Description :
In the Introduction and Theory page of 15th and 16th experiments of this lab, found that the content is same.

Actual Result :
In the Introduction and Theory page of 15th and 16th experiments of this lab, found that the content is same.

Environment :
OS: Windows 7, Ubuntu-16.04,Centos-6
Browsers:Firefox-42.0,Chrome-47.0,chromium-45.0
Bandwidth : 100Mbps
Hardware Configuration:8GBRAM ,
Processor:i5

Attachments:
![qa_oe_avlsi_i42](https://cloud.githubusercontent.com/assets/13479177/26667375/81be069a-46c2-11e7-8b11-5ca9e2cfd38c.png)

**Comment by sravanthimodepu** [comment]
Yes.. The 15th & 16th experiments belongs to the same content but the simulations are different. Found the same from the vlab.co.in page.

**Comment by BSravanthi** [comment]
Need to contact lab developer. The issue will be opened till we get the response.
